core/src/test/java/com/microsoft/applicationinsights/extensibility/initializer/docker/DockerContextInitializerTests.java

```diff
@@ -38,10 +39,8 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Test;
+
+import org.junit.*;
```

Comment:
Did you change Intellij settings? I think default is 99 imports before it uses '*'

Reply:
Yes. For tests, I allow * for `org.junit`, `org.hamcrest` and `org.mockito` and there are a few static ones too, e.g. `org.junit.Assert.*`. For prod code, I prefer to be explicit. For tests I like the splats for the test framework classes. Intellisense is usually quicker/better with its suggestions.

I think the checkstyle settings reflect this, but maybe not.

Reply:
How do you configure this differently in Intellij for test vs prod?

Reply:
I don't think you can. I have the count settings set to 500 (default is 99 but that didn't always work for me), plus rules for org.junit, Mockito, Matchers and Assert which always use `*`. These settings are under Editor/Code Style/Java/Imports